base/cvd/cuttlefish/host/libs/web/build_zip_name.cc

```diff
@@ -18,14 +18,31 @@
 #include <string>
 #include <variant>
 
+#include "third_party/android_cuttlefish/base/cvd/libbase/include/android-base/strings.h"
```

Comment:
This is an internal include path, in the bazel repository the path is only `android-base/strings.h`.